### ISSUE TITLE
Reverts door wire shocks back to  less than crit

### ISSUE THREAD

**Issue by aphielanis** [yogstation13/yogstation#1090]
Problem is someone coded door wire shocks to do crit which isn't fun and I don't support damage that does a one shot crit unless it's special circumstances or a high cap bomb.

 Probably the best solution is to revert the damage back to it's original code.

**Comment by X-TheDark** [comment]
Why, though. If an engineer messes and tries to decode the wires, he's an idiot for not having insulated gloves on.

If you do it as a Greytide McRandom, you're an idiot for sticking your big fat multitool into the door, you're not an engineer, hacking doors is not part of your job.

If you do it as Antag McBurdermoner, well...that's actually a valid point I can somewhat concede to, since you'd need to break into engineering with an emag to get the insulated gloves, which defeats the purpose since you have an emag.

Also, wearing most gloves protects against 50% of the shock damage AND fully output SMES only give 800k power, which translates to 80/+-5 damage, so again, someone would need to wire singulo into the grid, which gives the function an actual drawback, like the PR was supposed to do.

In summary, this screws over antags somewhat, I guess. Greytiders have no place to be hacking doors in the first place, if you want to do it, you'll have to take the risk of being insta-fried.

**Comment by puremanbird** [comment]
X-TheDark, all of your points are valid except the part where you actually play the game and experience this. I watched a single shocked door in science kill five people. Its not fun or rewarding to touch a door and instantly be dead.

**Comment by X-TheDark** [comment]
Oh...doors being shocked by someone else and people trying to open them...that's something I haven't actually considered...

**Comment by ShadowDeath6** [comment]
Can there be a set cap that can be bypassed by something such as powergloves

**Comment by aphielanis** [comment]
I don't think I am talking about door shocks themselves, I think that's governed by how much power is in the grid but if they do default crit regardless of the power grid then I am against it too.

 I am talking about the wire in the door that will shock people if accidentally probed or wirecut.

**Comment by AdamElTablawy** [comment]
Deal with it, things are deadly. There won't be a change.

A door is shocked? Get gloves. Ai rogue? Don't use airlocks. Someone wired the engine into the grid? Think twice before using a door. A door is hacked? Don't fucking jam your cock into it like a moron.

Anything and everything will kill you. Random events and incompetence included. Deal.

**Comment by ShadowDeath6** [comment]
>don't use airlocks

Lmao

**Comment by puremanbird** [comment]
Door is shocked? get gloves. after someone defibs your smoking corpse because you instantly died. Think twice before using a door. But dont carry tools unless your job needs them because thats powergaming. Yeah, it just doesnt work adam.
